.github/workflows/release.yml

```diff
@@ -151,9 +151,9 @@ jobs:
         uses: ncipollo/release-action@v1
         with:
           tag: ${{ env.new_tag }}
-          name: Helios Vortex ${{ env.new_version }}
+          name: Helios Engine ${{ env.new_version }}
```

Comment:
Probably just want to call it Helios? This is the actual 'Helios' binary, not the engine.
Same 2 lines below